dataflows/stretch_zmq_streaming.toml

```diff
@@ -0,0 +1,80 @@
+extra_modules = ["teleop.gopro_node", "teleop.webcam_node", "teleop.stretch_node", "teleop.video_streaming_node"]
+redis_url = "redis://172.26.57.91:6379/0"
+
+# GCP redis
+# redis_url = "redis://35.236.217.22:6379/0"
+
+[[nodes]]
+node_name = "tick"
+node_class = "tick"
+
+# [nodes.node_args]
+# millis_intervals = [20, 33]
+
+# [[nodes]]
+# node_name = "gopro"
+# node_class = "gopro"
+
+# [nodes.node_args]
+# input_tick_channel = "tick/millis/20"
+# output_channel = "wrist_cam"
+
+[[nodes]]
+node_name = "webcam"
+node_class = "webcam"
+
+[nodes.node_args]
+input_tick_channel = "tick/millis/50"
+output_channel = "meta_2_head_cam"
+webcam_name = "/dev/video6"
+width_resize_factor = 0.1
+height_resize_factor = 0.1
+
+[[nodes]]
+node_name = "video_streaming"
+node_class = "head_video_streamer"
+
+[nodes.node_args]
+input_channel = "meta_2_head_cam"
+output_channel = "head_video_status"
+quest_controller_ip = "172.26.172.110"
```

Comment:
Maybe change it to <your_quest_ip>

Reply:
We have removed the commented lines and update the redis and quest ip to placeholder in the following commit